app/src/data/schemas/searchSchema.ts

```diff
@@ -0,0 +1,77 @@
+const searchSchema = {
+  // Sort fields
+  // Filter fields
+  // Keyword included in search
+  response: {
+    headers: {
+      status: "200",
+      code: "200",
+      message: "some copy",
+    },
+    // Pagination fields
+    numResults: "6",
+    page: "1",
+    perPage: "40",
+    sort: "title DESC",
+    collections: [
```

Comment:
questions I have following the slack thread linked in the description from 12/5/2024...

if we're expecting collections, containers, and items to be returned in the paginated list of cards, should the array of objects be named "results" and include all three?

in other words, do we need to separate the objects into separate arrays by type?

Reply:
ie. results: [{}]
where each object has a field like "typeOfRecord" that is either "collection" "sub-collection" or "item"
copy is not final here but you get the idea